test/adapter/json_api/resource_meta_test.rb

```diff
@@ -31,11 +31,22 @@ class MetaBlockPostEmptyStringSerializer < ActiveModel::Serializer
             end
           end
 
-          def setup
+          class MetaBlockWithParameterSerializer < ActiveModel::Serializer
+            attributes :id
+            meta do |serializer|
+              { present: serializer.present? }
```

Comment:
I'm not sure what you are talking about. As far as I can see, in the reflection we `instance_exec` it the same way without any serializer method (https://github.com/rails-api/active_model_serializers/blob/ec045a5388a983f45a53726b5f9dd4b3ddbf0abd/lib/active_model/serializer/reflection.rb#L76). Maybe there is something I'm missing?